src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -326,3 +360,105 @@ def createTransferDoc(self, reqName, transferRecords):
         msg += "Error: %s" % resp
         self.logger.error(msg)
         return False
+
+
+    def _getStorageQuota(self):
```

Comment:
This method is not thread safe since it redefines member data self.nodeUsage all the time. I don't see **ANY** reason why you should keep it this way since you can easily convert to return local copy of nodeUsage dict to upstream caller and I suggest to do this way.